package.json

```diff
@@ -24,122 +24,123 @@
-    "start-prod": "tt-serve --env.disableHotReloading -p",
-    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
-    "test": "npm run lint && npm run jest && npm run wdio",
-    "wdio-default": "wdio",
-    "wdio-lowlight": "NODE_OPTIONS=--max-old-space-size=2048 THEME=clinical-lowlight-theme wdio",
-    "wdio-fusion": "THEME=orion-fusion-theme wdio",
-    "wdio": "npm run wdio-default && npm run wdio-lowlight && npm run wdio-fusion",
-    "version": "lerna version --no-git-tag-version && node ./scripts/changelog-updater/index.js"
+  "dependencies": {
+    "d3": "^5.16.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.5.0",
     "@babel/core": "^7.5.0",
     "@babel/plugin-proposal-object-rest-spread": "^7.5.0",
     "@babel/plugin-transform-object-assign": "^7.2.0",
     "@babel/plugin-transform-runtime": "^7.5.0",
-    "@babel/preset-env": "^7.5.0",
+    "@babel/polyfill": "^7.10.4",
+    "@babel/preset-env": "^7.10.4",
     "@babel/preset-react": "^7.0.0",
-    "@cerner/eslint-config-terra": "^4.0.0",
+    "@babel/runtime": "^7.10.4",
     "@cerner/browserslist-config-terra": "^3.0.0",
+    "@cerner/eslint-config-terra": "^4.0.0",
     "babel-eslint": "^10.1.0",
     "babel-jest": "^24.8.0",
+    "babel-loader": "^8.1.0",
+    "babel-plugin-istanbul": "^6.0.0",
+    "babel-plugin-minify-replace": "^0.5.0",
+    "babel-preset-minify": "^0.5.1",
     "check-installed-dependencies": "^1.0.0",
+    "classnames": "^2.2.5",
+    "commander": "^5.1.0",
     "core-js": "^3.1.3",
+    "css-loader": "^3.6.0",
+    "cssnano": "^4.1.10",
     "danger": "^9.1.0",
+    "docdash": "^1.2.0",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "enzyme-to-json": "^3.2.2",
     "eslint": "^6.1.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsdoc": "^24.0.6",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
     "glob": "^7.1.1",
+    "html-webpack-plugin": "^4.3.0",
+    "husky": "^4.3.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^24.8.0",
-    "lerna": "^3.16.4",
+    "jsdoc": "^3.6.4",
+    "lerna": "^3.22.1",
     "link-parent-bin": "^1.0.0",
+    "loader-utils": "^1.4.0",
+    "mini-css-extract-plugin": "^0.9.0",
+    "one-cerner-style-icons": "github:cerner/one-cerner-style-icons",
+    "postcss-loader": "^3.0.0",
+    "prettier": "^2.0.5",
+    "prop-types": "^15.5.8",
     "raf": "^3.4.1",
     "react": "^16.8.5",
     "react-dom": "^16.8.5",
     "react-test-renderer": "^16.4.2",
     "regenerator-runtime": "^0.13.2",
-    "stylelint": "^11.0.0",
+    "style-loader": "^2.0.0",
     "stylelint-config-terra": "^3.2.0",
     "terra-aggregate-translations": "^1.0.0",
     "terra-dev-site": "^6.5.0",
     "terra-disclosure-manager": "^4.9.0",
     "terra-enzyme-intl": "^3.0.0",
+    "terra-theme-context": "^1.0.0",
```

Comment:
this should be an actual dependency of the terra-graphs package, not a dev dependency.